client/app/services/vms.service.js

```diff
@@ -143,7 +143,7 @@ export function VmsService (CollectionsApi, RBAC) {
     return CollectionsApi.post(collection + '/' + vmId + '/snapshots/', null, {}, options)
   }
 
-  function createSnapshots (vmId, data) {
+  function createSnapshots (vmId,yarn install data) {
```

Comment:
Uhhh....what is that????

Reply:
its a feature, i don't know what our talking about